README.md

```diff
@@ -7,7 +7,7 @@
 [![Jenkins Plugin Installs](https://img.shields.io/jenkins/plugin/i/configuration-as-code.svg?color=blue)](https://plugins.jenkins.io/configuration-as-code)
 [![Gitter](https://badges.gitter.im/jenkinsci/configuration-as-code-plugin.svg)](https://gitter.im/jenkinsci/configuration-as-code-plugin)
 
-<img src="plugin/src/main/webapp/img/logo-head.svg" width="192">
+<img src="plugin/src/main/resources/images/symbols/logo.svg" width="192">
```

Comment:
I think we should keep colour in the README it doesn't look that great as-is on dark at least:

![image](https://github.com/user-attachments/assets/88e17288-a64d-4e5e-8149-c64d6d6d847a)

Reply:
Sure, the `currentColor` is only honered if the SVG is added inline. I can keep the color logo only for the doc